src/main/java/uk/gov/ch/model/officer/active/ActiveOfficerDetails.java

```diff
@@ -55,6 +57,9 @@ public void setServiceAddressPostCode(String serviceAddressPostCode) {
     }
 
     public String getUraLine1() {
+        if (getSecureIndicator().equals("Y")) {
```

Comment:
rather than have the 4 getSecureIndicator().equals("Y") checks, create one method isSecureOfficer that returns boolean and do the getSecureIndicator().equals("Y") check in there. then it will read better as well,

eg if(isSecureOfficer()) {
   return null;
} 
return uraLine1;